app/javascript/providers/SearchSelectionProvider.js

```diff
@@ -10,6 +11,31 @@ export const SearchSelectionContext = React.createContext({
   performSearch: undefined
 })
 
+/** returns the currently selected filter ids for the given facet object */
+export function getSelectionForFacet(facet, selectionContext) {
+  let selection = []
+  if (selectionContext.facets[facet.id]) {
+    selection = selectionContext.facets[facet.id]
+  }
+  return selection
+}
+
+/** returns whether or not the current selection for the given facet object is different
+  * than what the most recently performed search was */
+export function isFacetApplicable(facet, selectionContext, searchContext) {
+  const selection = getSelectionForFacet(facet, selectionContext)
+  const appliedSelection = getAppliedParamsForFacet(facet, searchContext)
+  const isSelectionValid = facet.type != 'number' ||
```

Comment:
Strict equality, for consistency with comparison for positive case (`===` and `!==`).

```suggestion
  const isSelectionValid = facet.type !== 'number' ||
```

Reply:
good catch